shared/test/metabase/driver/common/parameters/operators_test.cljc

```diff
@@ -43,23 +43,34 @@
                                      {:source-field 5}]]
                            :value ["foo"]})
            [:does-not-contain [:field 26 {:source-field 5}] "foo"])))
-  (testing "arity errors"
-    (letfn [(f [op values]
-              (try
-                (ops/to-clause {:type op
-                                :target [:dimension
-                                         [:field
-                                          26
-                                          {:source-field 5}]]
-                                :value values})
-                (is false "Did not throw")
-                (catch Exception e
-                  (ex-data e))))]
-      (doseq [[op values] [[:string/starts-with ["a" "b"]]
-                           [:number/between [1]]
-                           [:number/between [1 2 3]]]]
-        (is (schema= {:param-type (s/eq op)
-                      :param-value (s/eq values)
-                      :field-id s/Any
-                      :type (s/eq qp.error-type/invalid-parameter)}
-                     (f op values)))))))
+  #?(:clj
```

Comment:
clj only because i didn't feel like porting `schema=` at the moment. No idea if that's actually a problem. Just another rabbit hole i didn't feel like going down.